en/ch06.md

```diff
@@ -39,10 +56,12 @@ Use a remote node, do not download the blockchain. Check the ['About remote node
 (3) **Wallet path:** Where the wallet is located on your computer.  
 (4) **Wallet creation height:** Change the block height that a wallet rescan will go back to.  
 (5) **Wallet log path:** Where the logs for this wallet will be saved.  
-(6) **Copy to clipboard:** Copy all these informations to the clipboard.  
+(6) **Wallet mode:** Display the wallet mode chosen during the [wizard](#choose-wallet-mode).  
+(7) **Copy to clipboard:** Copy all these informations to the clipboard.  
```

Comment:
"Copy all these informations to the clipboard" -> "Copy all information to clipboard"

Reply:
ok